op-deployer/justfile

```
@@ -7,14 +7,9 @@ build-contracts:
 test args='./...': build-contracts
     go test -v {{args}}
 
-download-artifacts checksum outfile:
-    curl -o {{outfile}} -L https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-{{checksum}}.tar.gz
-
-calculate-artifacts-hash checksum:
-    just download-artifacts {{checksum}} /tmp/artifact.tgz
-    sha256sum /tmp/artifact.tgz
-    rm /tmp/artifact.tgz
-
+copy-contract-artifacts:
+    rm -f ./pkg/deployer/artifacts/forge-artifacts/artifacts.tgz
+    tar -cvzf ./pkg/deployer/artifacts/forge-artifacts/artifacts.tgz -C ../packages/contracts-bedrock/forge-artifacts --exclude="*.t.sol" .
```

```suggestion
    COPYFILE_DISABLE=1 tar -cvzf ./pkg/deployer/artifacts/forge-artifacts/artifacts.tgz -C ../packages/contracts-bedrock --exclude="forge-artifacts/*.t.sol" forge-artifacts
```
This way the archive will extract to a single `./forge-artifacts` directory inside the cache dir.

---

Why are we adding that extra layer?

---

1. Minimizes the chance of naming collisions in the cache.
2. Makes misconfiguration less painful: it's easier to clean up when the wrong cache dir is specified. This happened to me.